docs/operators-manual/day-2/check-ceph-status-disk-usage.md

```diff
@@ -73,3 +73,43 @@ For example:
     default.rgw.control        11   32      0 B        8      0 B      0    3.4 TiB
     default.rgw.meta           12    8    954 B        4   36 KiB      0    3.4 TiB
     default.rgw.buckets.index  13    8  2.2 MiB       11  6.6 MiB      0    3.4 TiB
+
+## Check Ceph OSD individual Disk Usage
+
+To view OSD utilization statistics use, `ceph osd df`
+
+For example:
+
+    # ceph osd df
+    ID  CLASS  WEIGHT   REWEIGHT  SIZE     RAW USE  DATA     OMAP    META     AVAIL    %USE  VAR   PGS  STATUS
+    2    ssd  0.87329   1.00000  894 GiB   77 GiB   75 GiB  17 KiB  1.2 GiB  818 GiB  8.57  1.00  227      up
+    0    ssd  0.87329   1.00000  894 GiB   77 GiB   75 GiB  17 KiB  1.2 GiB  818 GiB  8.57  1.00  227      up
+    1    ssd  0.87329   1.00000  894 GiB   77 GiB   75 GiB  17 KiB  1.2 GiB  818 GiB  8.57  1.00  227      up
+                          TOTAL  2.6 TiB  230 GiB  226 GiB  52 KiB  3.6 GiB  2.4 TiB  8.57                   
+    MIN/MAX VAR: 1.00/1.00  STDDEV: 0
+
+## Check Pool Statistics
```

Comment:
Remove this section since we have the `ceph df` earlier.

Also for each block that is important try to identify and add a section of what to watch out for.